src/styles/css/index.css

```diff
@@ -0,0 +1,116 @@
+* {
+  box-sizing: border-box; }
+
+body {
+  margin: 0; }
+
+.App {
+  font-family: 'helvetica neue', helvetica, sans-serif;
+  text-align: center; }
+
+.App-header {
+  background-size: cover;
+  background: url(../../assets/matt-lee-19500.jpg) bottom;
+  padding-bottom: 0;
+  padding-top: 30vh; }
+
+.App-header:after {
+  background: linear-gradient(to top right, rgba(76, 89, 115, 0.8) 40%, rgba(63, 188, 155, 0.8));
+  bottom: 10px;
+  content: "";
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 100%; }
+
+.header-catch-phrase {
+  color: #fff;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 40px;
+  position: relative;
+  z-index: 200; }
+
+.Categories-list {
+  align-content: space-around;
+  background: #eee;
+  color: #34495E;
+  display: flex;
+  flex-flow: row wrap;
+  font-family: 'Montserrat', sans-serif;
+  justify-content: space-around;
+  margin: 0 auto;
+  min-height: 35vh;
+  padding: 100px 70px;
+  position: relative;
+  text-transform: uppercase;
+  width: 100%;
+  z-index: 100; }
+
+.Category {
+  background: #fff;
+  border-radius: 5px;
+  box-shadow: 2px 2px 3px rgba(0, 0, 0, 0.05);
+  margin-bottom: 40px;
+  padding: 20px;
+  width: calc(20% - 30px); }
```

Comment:
if you add an additional min-width:200px property below the width property for .Category class here, it will let the category boxes wrap to next row nicely with shrinking/smaller browser window :) (currently the boxes continue to shrink without wrapping because they don't hit a hard width limit)